.github/ISSUE_TEMPLATE/bugReportForm.yml

```diff
@@ -0,0 +1,20 @@
+name: Bug Report
+description: File a bug report, we will respond to this thread with any questions.
+title: 'Bug: '
+labels: ['bug', 'triage']
+assignees: '@Azure/aks-atlanta'
+body:
+   - type: input
+     id: what-happened
+     attributes:
+        label: what happened?
+        description: also tell us what did you expect to happen?
+        placeholder: tell us what you see!
+     validations:
+        required: true
+   - type: input
+     id: logs
+     attributes:
+        label: relevant log output
+        description: run in debug mode for the most verbose logs. Please feel free to attach a screenshot of the logs
+        render: shell
```

Comment:
we also need to know the runner info (os and whether it was a custom runner). It's only not relevant for the freatureRequestForm.